examples/riot/coap_channel/Makefile

```diff
@@ -0,0 +1,28 @@
+# name of your application
+APPLICATION = lf-test
```

Comment:
Should we call it something more descriptive?

Reply:
Of course the makefile is super WIP right now I just make it compile and take care of that later :)